roles/common/tasks/monitoring.yml

```diff
@@ -207,6 +207,13 @@
 - name: log scanning caching dir
   file: path=/var/cache/check-log owner=sensu state=directory
 
+- name: bad memory alert
+  sensu_check: name=bad-memory plugin=check-mcelog.sh use_sudo=true
+               auto_resolve=false interval=20 occurrences=1
+               args='--criticality'
```

Comment:
This still needs updating @ulyssesbbg Will you be able to update it in time for our 3.1.0 release?